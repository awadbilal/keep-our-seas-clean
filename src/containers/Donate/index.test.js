import React from "react";
import renderer from "react-test-renderer";
import Donate from "./index";

it("renders correctly", () => {
  const tree = renderer.create(<Donate />).toJSON();
  expect(tree).toMatchSnapshot();
});
