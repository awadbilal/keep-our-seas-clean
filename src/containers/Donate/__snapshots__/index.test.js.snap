// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "minHeight": "84.6vh",
    }
  }
>
  <div
    className="main pt-5 pb-5 fdb-block container-fluid"
  >
    <div
      className="row"
    >
      <div
        className="col-md-8 offset-md-2 text-center col"
      >
        <h1
          className="title"
        >
          donate.title
        </h1>
      </div>
    </div>
  </div>
  <div
    className="pt-5 pb-5 fdb-block container"
  >
    <div
      className="align-items-center justify-content-center row"
    >
      <div
        className="px-4 col-lg-8 col-md-12 col-sm-12"
      >
        <form
          className=""
        >
          <div
            className="mb-3 row"
          >
            <div
              className="col"
            >
              <label
                className="form-label"
                htmlFor="formGridName"
              >
                form.name
              </label>
              <input
                className="form-control"
                id="formGridName"
                name="name"
                onChange={[Function]}
                placeholder="formPlaceholder.name"
                required={true}
                type="text"
                value=""
              />
            </div>
            <div
              className="col"
            >
              <label
                className="form-label"
                htmlFor="formGridSurname"
              >
                form.surname
              </label>
              <input
                className="form-control"
                id="formGridSurname"
                name="surname"
                onChange={[Function]}
                placeholder="formPlaceholder.surname"
                required={true}
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            className="mb-3 row"
          >
            <div
              className="col"
            >
              <label
                className="form-label"
                htmlFor="formGridEmail"
              >
                form.email
              </label>
              <input
                className="form-control"
                id="formGridEmail"
                name="email"
                onChange={[Function]}
                placeholder="formPlaceholder.email"
                required={true}
                type="email"
                value=""
              />
            </div>
            <div
              className="col"
            >
              <label
                className="form-label"
                htmlFor="formGridNumber"
              >
                form.number
              </label>
              <input
                className="form-control"
                id="formGridNumber"
                name="number"
                onChange={[Function]}
                placeholder="formPlaceholder.number"
                required={true}
                type="phone"
                value=""
              />
            </div>
          </div>
          <div
            className="mb-3"
          >
            <div
              className="mb-3 input-group"
            >
              <span
                className="input-group-text"
              >
                $
              </span>
              <span
                className="input-group-text"
              >
                Amount
              </span>
              <input
                aria-label="Dollar amount"
                className="form-control"
                id="exampleForm.ControlTextarea1"
                name="amount"
                onChange={[Function]}
                required="true"
                type="number"
                value=""
              />
            </div>
          </div>
          <button
            className="submitButton btn btn-primary"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            form.submit
          </button>
        </form>
      </div>
    </div>
  </div>
</div>
`;
